Makefile

```diff
@@ -68,9 +74,35 @@ publish-version:
 	  exit 1; \
 	fi
 
-tag:
-	$(GIT_TAG) --message v$(VERSION) v$(VERSION)
-	$(GIT_PUSH) origin refs/tags/v$(VERSION)
+publish-pip:
+	python setup.py sdist bdist_wheel
+	pip install twine
+	twine upload dist/*
```

Comment:
How are PyPi credentials set / injected?  Locally I have `~/.pypirc` set up

Reply:
`TWINE_USERNAME` and `TWINE_PASSWORD` credentials are set in the environment via the Circle CI project configuration.